### ISSUE TITLE
wlroots >=0.16.0

### ISSUE THREAD

**Issue by bda65hp** [project-repo/cagebreak#35]
Hi,
cagebreak is not my main compositor but i like it.
Since a few days, Arch Linux updated to wlroots 0.16.1 and cagebreak can't be used anymore without downgrading wlroots.
Do you plan to update to newer wlroots versions soon or do i need to downgrade some packages on my pc?

**Comment by project-repo** [comment]
Hi bda65p,

it's great to hear that you like cagebreak.

We are on it. At the moment this coincides with the preparations for
a release which was planned anyway. In the meantime you probably
won't get around downgrading cagebreak.

We will let you know once a viable version of cagebreak is ready on the
development branch (we will then offer you a PKGBUILD for the development
branch which you can use instead of the one on the AUR, in case you
prefer that) and you will probably be notified by github once we release.

cheers and sorry for the inconvenience
project-repo

**Comment by project-repo** [comment]
Hi bda65p,

cagebreak now builds with wlroots 0.16.1.

We have not yet done the usual quality assurance, but it should mostly
work if you check out the development branch.

You may use the attached PKGBUILD to do this automatically.

cheers
project-repo

```
# Maintainer: project-repo <archlinux-aur@project-repo.co>
pkgname=cagebreak
pkgver=2.0.0
pkgrel=1
pkgdesc='Tiling wayland compositor based on cage inspired by ratpoison'
arch=('x86_64')
url='https://github.com/project-repo/cagebreak'
license=('MIT')
depends=('wayland' 'libxkbcommon' 'wlroots<0.17.0' 'pango')
makedepends=('meson' 'ninja' 'scdoc' 'wayland-protocols')
optdepends=('wl-clipboard: clipboard support'
            'xorg-xwayland: x application support')
options=('!buildflags' '!strip')
conflicts=('cagebreak-bin')
source=("cagebreak::git+https://github.com/project-repo/cagebreak")
sha512sums=('SKIP')
build() {
	cd "$pkgname"
	git checkout development
	meson setup build --buildtype=release -Dxwayland=true -Dman-pages=true
	ninja -C build
}
package() {
	cd "$pkgname"
	mkdir -p "$pkgdir/usr/bin/"
	cp 'build/cagebreak' "$pkgdir/usr/bin/"
}
```

**Comment by bda65hp** [comment]
Thanks. I can use cagebreak now :)

**Comment by project-repo** [comment]
Hi bda65p,

cagebreak 2.0.0 is released now and compatible with wlroots 0.16.1.

I am closing this issue.

cheers
project-repo
